Lib/test/test_smtpnet.py

```diff
@@ -2,13 +2,16 @@
 from test import support
 from test.support import import_helper
 from test.support import socket_helper
+import os
 import smtplib
 import socket
 
 ssl = import_helper.import_module("ssl")
 
 support.requires("network")
 
+SMTP_TEST_SERVER = os.getenv('SMTP_TEST_SERVER', 'smtp.gmail.com')
```

Comment:
```suggestion
SMTP_TEST_SERVER = os.getenv('CPYTHON_TEST_SMTP_SERVER', 'smtp.gmail.com')
```

following precedent from https://github.com/python/cpython/blob/1aa8bbe62f27b564cf15e2aad591c62744354a4e/Lib/test/test_cppext/setup.py#L28-L29